contrib/completion/bash/docker

```diff
@@ -989,6 +989,11 @@ _docker_ps() {
 			COMPREPLY=( $( compgen -W "exited paused restarting running" -- "${cur#=}" ) )
 			return
 			;;
+		*image=*)
```

Comment:
oh waw I'm definitely not awake today.. This is supposed to be `ancestor`.. I might need some sleep :sweat: